core-processor/src/processing.rs

```diff
@@ -297,18 +312,8 @@ fn process_error(
     }
 
     if !dispatch.is_reply() && dispatch.kind() != DispatchKind::Signal {
-        let (err, err_payload) = case.to_reason_and_payload();
-
-        // Panic is impossible, unless error message is too large or [Payload] max size is too small.
-        let err_payload = err_payload.unwrap_or_else(|| {
-            let (_, err_payload) = case.to_reason_and_payload_str();
-            let err_msg =
-                format!("process_error: Error message is too big. Message id - {message_id}, error payload - {err_payload}",
-            );
-
-            log::error!("{err_msg}");
-            unreachable!("{err_msg}")
-        });
+        let err = case.to_reason();
+        let err_payload = case.to_payload();
```

Comment:
this can consume self in order to avoid panic buff clone

Reply:
`case` is used further